### ISSUE TITLE
Transactiondetails are different (Android / iOS) 

### ISSUE THREAD

**Issue by Espo02** [NemProject/NEMiOSApp#270]
The Transactiondetails are much better on Android.
It is more intuitive du slide the transaction to the middle.

Android:
![screenshot_20160809-210207](https://cloud.githubusercontent.com/assets/17567386/17529961/b2f3fc00-5e75-11e6-88c8-b458915186ac.png)

iOS:
![photo-2016-08-09-21-07-33_3022](https://cloud.githubusercontent.com/assets/17567386/17529966/b839ac82-5e75-11e6-9166-b84daf3937ca.PNG)

Espo: NDEFC3-H4VAIU-FQ5GFT-IK3IJR-4DGKF7-PEQKRW-LPZ5

**Comment by fyahfox** [comment]
I agree of course, since I always wanted that the 2 apps look as similar as possible.
But I guess this is not a priority for now.

**Comment by jabo38** [comment]
Bounty Paid Transaction hash: 9b533dc661baad9713c13771a4bfe620e6694fa73305cfcb8792d65deb1cdc8c
